ci-operator/step-registry/sandboxed-containers-operator/e2e/aws/sandboxed-containers-operator-e2e-aws-workflow.yaml

```diff
@@ -20,6 +20,23 @@ workflow:
     post:
     - ref: sandboxed-containers-operator-gather-must-gather
     - ref: cucushift-installer-wait
+    - as: sandboxed-containers-operator-delete-kataconfig
```

Comment:
Hi @ldoktor ! I think it's a good idea to try to delete the resources at the end. One question: the plan is to move this code to step-registry once we have the profile to azure as well?

Reply:
Should be for both :-)